cluster-autoscaler/processors/nodegroupset/compare_nodegroups.go

```diff
@@ -33,7 +33,7 @@ const (
 	MaxFreeDifferenceRatio = 0.05
 	// MaxMemoryDifferenceInKiloBytes describes how much memory
 	// capacity can differ but still be considered equal.
-	MaxMemoryDifferenceInKiloBytes = 128000
+	MaxMemoryDifferenceInKiloBytes = 256000
```

Comment:
Split the commit in PR to two:
 - one with change to MaxMemoryDifferenceInKiloBytes
 - the other with IsAwsNodeInfoSimilar

Reply:
Done